model_analyzer/config/generate/automatic_model_config_generator.py

```diff
@@ -116,19 +116,19 @@ def _get_curr_param_combo(self):
             return DEFAULT_CONFIG_PARAMS
 
         config = {
-            'dynamic_batching': {},
             'instance_group': [{
                 'count': self._curr_instance_count,
                 'kind': self._instance_kind
             }]
         }
 
-        if not self._sweep_max_batch_size_disabled:
+        if not self._max_batch_size_disabled:
             config['max_batch_size'] = self._curr_max_batch_size
+            config['dynamic_batching'] = {}
```

Comment:
What if the user doesn't want to enabled dynamic batching?
For example in the L0_config test - the first run has a batch size of 128, but no dynamic batching. And, then the second run enabled dynamic batching. Now aren't you going to produce the same config?

Reply:
The user has full control if they manually specify what they want to search over. If they don't provide us with anything, then the automatic search does what it can to find the best config -- which always involves turning on dynamic batching.

Yes, this could result in the default config being part of the search space. This already can happen and is fine.